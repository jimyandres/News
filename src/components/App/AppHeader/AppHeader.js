import React from 'react';
import logo from '../../../logo.svg';

const AppHeader = () =>
	<header className="App-header">
		<img src={logo} className="App-logo" alt="logo" />
		<h1 className="App-title">Welcome to Hacker News with React</h1>
	</header>;

export default AppHeader;
