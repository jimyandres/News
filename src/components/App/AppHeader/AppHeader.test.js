import React from 'react';
import ReactDOM from 'react-dom';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallowToJson } from 'enzyme-to-json';
import AppHeader from './AppHeader';

configure({ adapter: new Adapter() });

describe('AppHeader', () => {

	it('renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(<AppHeader />, div);
		ReactDOM.unmountComponentAtNode(div);
	});

	test('has a valid snapshot', () => {
		const component = shallow(<AppHeader />);
		expect(shallowToJson(component)).toMatchSnapshot();
	});
});
