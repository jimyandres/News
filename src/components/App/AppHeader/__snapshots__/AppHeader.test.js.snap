// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader has a valid snapshot 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
  <h1
    className="App-title"
  >
    Welcome to Hacker News with React
  </h1>
</header>
`;
